File: test/unit/code_parser/whole_tests/views/help_comments.vb
' Copyright (c) 2000 ActivePDF, Inc.
' ActivePDF Server 2009
' Example generated 01/01/2000 

Imports System

Public Class Examples
  Sub Example()
    ' Hello World!
    ' Hello .NET!
    ' Hello .NET!
    ' You can display a \ and also a " in the comment
    ' Process Complete
    WriteResults("Done!")
  End Sub
  
  ' Write output data
  Sub WriteResults(content As String)
    ' Choose where to write out results
  
    ' Debug output
    'System.Diagnostics.Debug.WriteLine("ActivePDF: * " + content)
  
    ' Console
    Console.WriteLine(content)
  
    ' Log file
    'Using tw = New System.IO.StreamWriter(AppDomain.CurrentDomain.BaseDirectory & "application.log", True)
    '   tw.WriteLine("[" + DateTime.Now.ToString("MM/dd/yyyy HH:mm:ss") + "]: => " + content)
    'End Using
  End Sub
End Class

File: test/unit/code_parser/whole_tests/sources/help_comments.rb
# Write out a comment
comment "Hello World!"

# Write out COM or .NET specific comments
comment :com => "Hello COM!"
comment :net => "Hello .NET!"

# Write out to both COM and .NET with one line of code
comment :com => "Hello COM!", :net => "Hello .NET!"

# Some characters need to be escaped in ruby such as \ and "
comment "You can display a \\ and also a \" in the comment"
